model/clusters_mgmt/v1/cluster_autoscaler_type.model

```diff
@@ -0,0 +1,41 @@
+/*
+Copyright (c) 2021 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Cluster-wide autoscaling configuration.
+struct ClusterAutoscaler {
+	// BalanceSimilarNodeGroups enables/disables the
+	// `--balance-similar-node-groups` cluster-autocaler feature.
+	// This feature will automatically identify node groups with
+	// the same instance type and the same set of labels and try
+	// to keep the respective sizes of those node groups balanced.
+	BalanceSimilarNodeGroups Boolean
+
+	// Enables/Disables `--skip-nodes-with-local-storage` CA feature flag. If true cluster autoscaler will never delete nodes with pods with local storage, e.g. EmptyDir or HostPath. true by default at autoscaler
+	SkipNodesWithLocalStorage Boolean
+
+	// Sets the autoscaler log level.
+	// Default value is 1, level 4 is recommended for DEBUGGING and level 6 will enable almost everything.
+	LogVerbosity Integer
```

Comment:
Do we want this to remain an int we propagate to AS, or maybe type this `LogVerbosityDebug` LogVerbosityInfo` etc.?

Reply:
For flexibility purposes and for conforming with kubernetes schema I'd say we should leave it generic. Who knows what other log-level they'll add in the future